lookbook/docs/patterns/13-feedback-dialog-and-feedback-message.md.erb

```diff
@@ -0,0 +1,65 @@
+The Feedback Dialog is used to provide the user feedback on a multi-step process (or a complex process). Unlike the `Banner`/`Flash`, the Feedback dialog is intrusive and demands user attention. In some cases, it is the final step of a multi-step process, although it can also be triggered independently.
+
+The Feedback Message is a sub-component (a variation of `Blankslate`) that is contained in the Feedback Dialog, but can also be used separately.
+
+### Overview
+
+[Live Example]
+
+### Anatomy
+
+The Feedback Dialog is made up of three parts. The are variations of existing Primer components and the third is a blanket area for any additional content:
+
+- **Feedback Dialog** is the dialog modal, itself a variation of the `Dialog`
+- **Feedback Message** is what is contained in the dialog, itself a variation of 'Blankslate'.
+- **Additional content** can be placed below the Feedback Message when you need to include other important elements like a checkbox, more text or a banner.
+
+Both of these inherit all the properties of their original Primer components and as such can be adapted to a number of different use cases, as illustrated below. 
```

Comment:
This whole section is somehow weird. First you are talking about three parts, in the last sentence it is suddenly "Both" . Further, saying that the FeedbackDialog consists of three parts and then one of them is the `FeedbackDialog` does not really make sense. 

I would structure it differently:

```suggestion
The Feedback Dialog is a variation of a normal Primer Dialog, with two possible slots:

- **Feedback Message** (mandatory) is what is contained in the dialog, itself a variation of 'Blankslate'.
- **Additional content** (optional) can be placed below the Feedback Message when you need to include other important elements like a checkbox, more text or a banner.
```